earthmover/earthmover.py

```diff
@@ -548,14 +549,20 @@ def build_package_graph(self, root_node: str, package_subgraph: Graph, packages_
 
     def clean(self):
         """
-        Removes local artifacts created by `earthmover run`
+        Removes local artifacts created by `earthmover run` and `earthmover compile`
         :return:
         """
 
         was_noop = True
-        if os.path.isdir(self.state_configs['output_dir']):
-            shutil.rmtree(self.state_configs['output_dir'], ignore_errors = True)
-            was_noop = False
+        output_dir = Path(self.state_configs['output_dir'])
+        if output_dir.is_dir():
+            if Path(output_dir / "earthmover.yaml").is_file() or Path(output_dir / "earthmover.yml").is_file():
```

Comment:
Thanks for the quick fix, @johncmerfeld !

This is nit-picky and not super important, but curious why you added the `Path` import above and construct the paths with `Path(a / b / c)`, rather than using `os.path.join(a, b, c)` [like we do elsewhere](https://github.com/edanalytics/earthmover/blob/3abb434801820c14e33e25c25e26f533d69f289a/earthmover/earthmover.py#L69) in earthmover?

Reply:
No reason in particular; I just like pathlib I guess. Happy to change if you prefer!

Reply:
Curious if @jayckaiser has any preference here.

I have no real reason to prefer one method over the other, but I do prefer consistency.

Reply:
@tomreitz I changed it to `os`. I hear you on the consistency

Reply:
Thanks!